### DISCUSSION TITLE
ER diagram

### DISCUSSION THREAD

**Discussion by overlapped** [sqlitebrowser/sqlitebrowser#2694] [Ideas]
Hi everyone!

So, what about develop plugin/extension for SQLite Browser provide something like ER diagram and design case tool?

**Comment by scottfurry** [comment]
Not a new subject
See #913  #1962 #2034

**Comment by chrisjlocke** [comment]
Or, "feel free to contribute if you can".  DB4S is an open-source project, so you're more than welcome to contribute.
